db/db-populate.js

```diff
@@ -0,0 +1,9 @@
+var description = require('../tutorials/descriptions')
+var sources = require('../tutorials/sources').startingLinksSources
+const tutorialDB = require('../db/tutorial-queries')
+
+module.exports.populateDB = function () {
+  for (let i = 0; i < 2; i++) {
```

Comment:
I think this the code you run when the table is first set up? Maybe make a note on the role of this.